app/TracingStrategyAssets.ts

```diff
@@ -3,7 +3,28 @@ import { useTranslation } from 'react-i18next';
 import { Images } from './assets';
 import { isGPS } from './COVIDSafePathsConfig';
 
-export function useAssets() {
+export type TracingStrategyAssetMap = {
+  onboarding2Background: string;
```

Comment:
@johnschoeman not sure if this is remotely idiomatic; `onboarding2Background` for instance is an image - is it encoded as a string or is a different type needed here?

Reply:
Yeah, noticed this too.  I think what you have here is fine.  but i think the ideal solution is to not have mixed value types, its a bit odd to have the strings and images mixed as such. so less of a typescript issue and more of the api could be changed to separate the text and the images.

a note, you could have done something like:
```
export function useAssets(): Record<string, string>
```

and ts should infer the correct names for you on the object

or even something like

```
export function useAssets(): Record<string, string | Image>
```
where `Image` is whatever the image is supposed to be.